features/step_definitions/filtering_steps.rb

```diff
@@ -738,8 +738,19 @@
   expect(page).to have_button("Create subscription")
 end
 
-Then("I cannot select any filters") do
-  find("input[name='filter[facet_groups][]']", visible: false)
+And("I can see the business finder filters") do
+  # The facet_group value for the business finder is not optional so should exist
+  # as a hidden input
+  expect(page).to have_css("input[name='filter[facet_groups][]']", visible: false)
+  expect(page).to have_css("input[value='52435175-82ed-4a04-adef-74c0199d0f46']", visible: false)
```

Comment:
Does this refer to the same element? In which case, may be better to combine, something like:

```ruby
expect(page).to have_css("input[name='filter[facet_groups][]'][value='52435175-82ed-4a04-adef-74c0199d0f46']", visible: false)
```